app/frontend/javascripts/classes/StoreManagerMixin.js

```diff
@@ -333,6 +334,15 @@ export const mixin = {
           }
         })
         .catch((e) => {
+          const body = document.querySelector('body');
+          const modalWindow = new ModalWindow(body, 'Some error!');
```

Comment:
ありがとうございます！
`some error` は、例として使っていましたので、実際にエラーの内容を表示されたらいいのではないかと思っています：
`catch((e) => ...)`  の`e` には　[Error](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Error/message) のオブジェクトが入りますので、 e.message にはそのエラーのメッセージが文字列として入っています。

```suggestion
          const modalWindow = new ModalWindow(body, e.message);
```